[TASK] Improve slug eval options (#280)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Properties/SlugEval.rst.txt

    Keywords:

    unique
        Evaluate if a record is unique in the whole TYPO3 installation (specific to a language).

>         This option is recommended as it allows to show any record stored inside other site.

Reviewer: ```suggestion
        This option is recommended as it allows to show any record stored inside other sites.
```

    Keywords:

    unique
        Evaluate if a record is unique in the whole TYPO3 installation (specific to a language).

>         This option is recommended as it allowes to show any record at any other possible site.

Reviewer: ```suggestion
        This option is recommended as it allows to show any record stored inside other site.
```

>         The only downside is that it is not possible to have the same identical slug on multiple sites.

Reviewer: ```suggestion
        The only downside is that it is not possible to have the same slug on multiple sites.
```

    uniqueInSite
        Evaluate if a record is unique in a site (specific to a language).

        .. warning::

>               Be aware that using this option makes it impossible to show records persisted in a page of one site
>               on pages of another site!

Reviewer: ```suggestion
              Be aware that using this option makes it impossible to show records stored inside other sites
```
